### ISSUE TITLE
Possible bug

### ISSUE THREAD

**Issue by ash567** [GuessWhatGame/guesswhat#21]
In the following files, while passing the config to optimizer, config["optimizer"] must be passed.

"
guesswhat/src/guesswhat/train/train_guesser.py:
   ..
   78      # Build Optimizer
   79      logger.info('Building optimizer..')
   80:     optimizer, outputs = create_optimizer(network, config)
   81  
   82      ###############################

guesswhat/src/guesswhat/train/train_oracle.py:

   85      # Build Optimizer
   86      logger.info('Building optimizer..')
   87:     optimizer, outputs = create_optimizer(network, config, finetune=finetune)
   88  
   89      ###############################

guesswhat/src/guesswhat/train/train_qgen_reinforce.py:

  124:     pg_optimize, _ = create_optimizer(qgen_network, loop_config,
  125                                     var_list=pg_variables,
  126                                     optim_cst=tf.train.GradientDescentOptimizer,
  127                                     loss=qgen_network.policy_gradient_loss)
  128:     baseline_optimize, _= create_optimizer(qgen_network, loop_config,
  129                                          var_list=baseline_variables,
  130                                          optim_cst=tf.train.GradientDescentOptimizer,

guesswhat/src/guesswhat/train/train_qgen_supervised.py:

   79      # Build Optimizer
   80      logger.info('Building optimizer..')
   81:     optimizer, outputs = create_optimizer(network, config)
   82  
   83      ###############################
"

**Comment by fstrub95** [comment]
Thx for pointing it out. The fix was not pushed :(
https://github.com/GuessWhatGame/guesswhat/commit/5ae64961bde2460718191b1807acd5a79fb52bb6
